src/components/edit-user.html

```diff
@@ -1,95 +1,116 @@
+      }
 
-            .border {
-                border: #666 1px solid;
-                width: 20%;
-                margin: auto;
-                margin-top: 25px;
-                padding-bottom: 25px;
-            }
+      input[type=text],
+      select {
+        width: 100%;
+        padding: 12px 20px;
+        margin: 8px 0;
+        display: inline-block;
+        border: 1px solid #87CEFA;
+        border-radius: 4px;
+        box-sizing: border-box;
+      }
 
-            .button {
-                background-color: #87CEFA;
-                border: none;
-                color: white;
-                padding: 5px 12px;
-                text-align: center;
-                text-decoration: none;
-                display: block;
-                margin: auto;
-                font-size: 16px;
-                margin-top: 25px;
-            }
+      div {
+        border-radius: 5px;
+        background-color: #f2f2f2;
+        padding: 20px;
+      }
 
-            button:hover {
-                background-color: rgb(196, 226, 245);
-            }
-        </style>
-
-        <h1 class="div-center">Edit User</h1>
-        <div>
-            <div class="container">
-                <div class="form-border">
-                    <form>
-                        <label for="fname">First Name</label>
-                        <input type="text" id="fname" name="firstname" placeholder="Your name..">
-
-                        <label for="lname">Last Name</label>
-                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
-
-                        <label for="phone">Phone</label>
-                        <input type="text" id="phone" name="phone" placeholder="Your phone number..">
-
-                        <label for="email">Email</label>
-                        <input type="text" id="email" name="email" placeholder="Your email..">
-                        <div>
-                            <button class="button">Edit</button>
-                        </div>
-                    </form>
-                </div>
-            </div>
+      .border {
+        border: #666 1px solid;
+        width: 20%;
+        margin: auto;
+        margin-top: 25px;
+        padding-bottom: 25px;
+      }
+
+      .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+    </style>
+
+    <h1 class="div-center">Edit User</h1>
+    <div>
+      <div class="container">
+        <div class="form-border">
+          <form>
+            <label for="fname">Enter User Id</label>
+            <input type="text" placeholder="User Id for edit and delete functions tests.." value="{{user.userId::input}}">
```

Comment:
Have this for the user to be able to choose which user they want to edit or delete.

Reply:
Ok got this wrong. However...
I think in the end, when you polish your project, you're not going to be super happy with this implementation. Maybe not, but that's my feeling.

For instance: **Remembering an id, navigating to edit user, inputing remembered id** has a _much_ greater chance for error than something like **Click edit button on user profile, user info pre-populates edit user component**. The latter method has virtually _no_ error potential (with the exception of a mis-click).

Reply:
I agree with ya, that would be a better way to go about it. I'll go ahead and get that going.

Reply:
Something like this?

![editUserRedesign](https://user-images.githubusercontent.com/29979289/62232458-e2e51200-b38b-11e9-9a35-53ed7ee7ab12.jpg)

Reply:
Still thinking of how to avoid issues with the 'edit field' part. For instance, if they want to edit 'firstName' and don't case the prop name right it'll add a second 'firstname' prop.

Reply:
> Something like this?

Yessir. The only comment I would have is I would have the expand/collapse button down with the Edit User button. But that's just what I'd expect

Reply:
Sounds good. Thanks!

Reply:
> For instance, if they want to edit 'firstName' and don't case the prop name right it'll add a second 'firstname' prop.

yeah that's just going to be an issue with any kind of "create a property" type logic, hence the static inputs that _only_ change a single prop.

you could modify the property name, once inputed, to camel case. check if the property is already created and notify the user the property already exist. I don't know how much of a rabbit hole that would end up being, but a potential solution none the less.